components/SponsorTable.tsx

```diff
@@ -31,9 +34,6 @@ const SponsorTable = ({isVisible} : TSponsorTable) => {
                     <a href="https://pisa.dev/" target="_blank" rel="noreferrer">
                     <Image width={150} height={150} src="/pisa-dev_logo.svg" alt="PisaDev" />
                     </a>
-                    <a href="https://firenze.dev/" target="_blank" rel="noreferrer">
-                    <Image width={100} height={100} src="/firenze-dev_logo.svg" alt="Firenzedev" />
-                    </a>
```

Comment:
I think we can wait a little bit before removing it definitively.

Reply:
I'm not agree, when we know their intention we add their logo again

Reply:
I agree on Nic